docs/rest.md

```diff
@@ -686,6 +686,28 @@ Generally the UUID of a background operation you are interested in.
 * Operation: sync
 * Return: an object with the created username and the ssh keys imported.
 
+#### Input
+
+A javascript object with the following keys:
+* email: the email of the user to create
+* sudoers: if true adds "sudo" access to the created user
+* known: use the local system-user assertions to create the user
```

Comment:
Hm, we don't have any documenation about assertions afaict. I crated a stub docs/assertions.md that contains basic details about the existing assertions, that should be a start at least.